dateparser/custom_lang_detect/map_languages.py

```diff
@@ -0,0 +1,9 @@
+from dateparser_data.language_maps import languages_map
+
+
+def map_languages(language_codes):
```

Comment:
Is language mapping going to be independent on the library?

Reply:
Sorry, I could not understand.
The use of this function to check if the detected language is supported by dateparser and if yes return the different supported options in the library like if `"az"` is passes it should return `"az", "az-Cyrl", "az-Latn".`


@lopuhin

Reply:
I meant, can it happen that different language detection libraries output different language identifiers and so could need different mappings?

Reply:
We and other language detection we plan to support library (fasttext and langdetct) are using ISO-629 so I have kept the base support for the same. However, with your point, we can discuss that at our next meeting.

Reply:
Sounds good, thanks 👍